_component_design/cards.md

```diff
@@ -99,36 +64,36 @@ Small and large spacing values can be used for card variations but should not be
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Spacing
+### Composition
 
-Each section of a card uses medium container spacing, 20px, by default.
+The card pattern is composed of the following three elements:
 
-Small and large spacing values can be used for card variations but should not be mixed and matched within a layout or card.
+- **Header (optional):** The header establishes the context for a card and contains icons (such as a cog) that allow the user to take action.
+- **Body (required):** The body contains the primary content associated with the card. The body section details the underlying object represented by the card.
+- **Footer (optional):** The footer provides calls to action that a user can take.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [class:"image bg-light border"] %}
- ![]({{site.url}}/assets/images/components/content-areas/cards/card-spacing.png){:width="611px"}
+ ![]({{site.url}}/assets/images/components/content-areas/cards/card-default.png){:width="611px"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
 </section>
 
-## Layout
-
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Grid
+### Style
 
-Cards are applied to the helix-grid and have a variable width and height based on content, current view-port, and use case.  
+Cards are applied to the Helix Grid and have a variable width and height based on content, current view-port, and use case.  
 
-- Default cards are styled with a #FFFFFF background and should only go on darker backgrounds.
-- For more information see [the Helix-ui Grid](https://rackerlabs.github.io/helix-ui/components/grid/)
+- Default cards are styled with an #FFFFFF background and should only go on darker backgrounds.
+- For more information see the [Helix UI Grid](https://rackerlabs.github.io/helix-ui/components/grid/).
```

Comment:
Out of curiosity, why are we differentiating between the Helix Grid and the Helix UI Grid? They are the same thing, and I think the typical user looking at this will be looking at the designer-focused grid page rather than the developer-focused grid page.